hazelcast/include/hazelcast/client/HazelcastClient.h

```diff
@@ -512,6 +512,26 @@ namespace hazelcast {
             */
             IdGenerator getIdGenerator(const std::string& name);
 
+            /**
+             * Creates a cluster-wide unique ID generator. Generated IDs are {@code long} primitive values
```

Comment:
From this comment it seems like every time this method is called a new flake ID generator will be created, but I think if one already exists with the given name, it wont be created right? If that is the case, I think we should mention about that here.

Reply:
This is the same javadoc for the Java client. if needed, Java client javadoc also needs to be changed. I changed it to: "Returns a generator that creates a cluster-wide unique IDs. "